tools/eslint-rules/rules/use-getLoadable-and-getValue-to-get-atoms.ts

```diff
@@ -0,0 +1,59 @@
+import { ESLintUtils } from '@typescript-eslint/utils';
+
+// NOTE: The rule will be available in ESLint configs as "@nx/workspace-usage-getLoadable-and-getValue-to-get-atoms"
+export const RULE_NAME = 'use-getLoadable-and-getValue-to-get-atoms';
+
+export const rule = ESLintUtils.RuleCreator(() => __filename)({
+  name: RULE_NAME,
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Ensure you are using getLoadable and getValue',
+      recommended: 'recommended',
+    },
+    fixable: 'code',
+    schema: [],
+    messages: {
+      invalidWayToGetAtoms:
+        "Expected usage '{{ expectedWayToGetAtoms }}' but found '{{ actualWayToGetAtoms }}'",
+    },
+  },
+  defaultOptions: [],
+  create: (context) => ({
+    AwaitExpression: (node) => {
+      const { argument: arg }: any = node;
+      if (
+        arg.callee &&
+        arg.callee.type === 'MemberExpression' &&
+        arg.callee.object.name === 'snapshot' &&
+        arg.callee.property.name === 'getPromise' &&
+        arg.arguments &&
+        arg.arguments[0]
+      ) {
+        let actualWayToGetAtoms: string;
+        let expectedWayToGetAtoms: string;
+
+        if (arg.arguments[0].callee) {
```

Comment:
We should also check that we use `.getValue` because we can use many properties or functions : 

<img width="169" alt="image" src="https://github.com/twentyhq/twenty/assets/26528466/37214139-0b25-4f78-a7d3-a176df098124">

Reply:
Alright